src/eslint.ts

```diff
@@ -60,6 +60,11 @@ export interface EslintOptions {
    * @default undefined
    */
   readonly aliasExtensions?: string[];
+
+  /**
+   * Always try to resolve types under `<root>@types` directory even it doesn't contain any source code
+   */
+  readonly typescriptAlwaysTryTypes?: boolean;
```

Comment:
I think we use `ts` throughout.

```suggestion
  readonly tsAlwaysTryTypes?: boolean;
```

Reply:
Done.